SimplenoteUITests/SimplenoteUITestsSearch.swift

```diff
@@ -1,22 +1,43 @@
-
-let mechagodzillaNoteName = "Mechagodzilla"
-let mechagodzillaInfo = "Mechagodzilla (メカゴジラ, Mekagojira) is a fictional mecha character that first appeared in the 1974 film Godzilla vs. Mechagodzilla. In its debut appearance, Mechagodzilla is depicted as an extraterrestrial villain that confronts Godzilla. In subsequent iterations, Mechagodzilla is usually depicted as a man-made weapon designed to defend Japan from Godzilla. In all incarnations, the character is portrayed as a robotic doppelgänger with a vast array of weaponry, and along with King Ghidorah, is commonly considered to be an archenemy of Godzilla."
-
-let diacriticNoteName = "Diacritic"
-let diacriticInfo = "A diacritic (also diacritical mark, diacritical point, diacritical sign, or accent) is a glyph added to a letter or basic glyph. The term derives from the Ancient Greek διακριτικός (diakritikós, \"distinguishing\"), from διακρίνω (diakrī́nō, \"to distinguish\"). Some diacritical marks, such as the acute ( ´ ) and grave ( ` ), are often called accents. Examples are the diaereses in the borrowed French words naïve and Noël, which show that the vowel with the diaeresis mark is pronounced separately from the preceding vowel; the acute and grave accents, which can indicate that a final vowel is to be pronounced, as in saké and poetic breathèd; and the cedilla under the \"c\" in the borrowed French word façade, which shows it is pronounced /s/ rather than /k/. In other Latin-script alphabets, they may distinguish between homonyms, such as the French là (\"there\") versus la (\"the\") that are both pronounced /la/. In Gaelic type, a dot over a consonant indicates lenition of the consonant in question."
+// swiftlint:disable line_length
+let godzilla = NoteData(
+    name: "Godzilla",
+    content: "Godzilla (Japanese: ゴジラ, Hepburn: Gojira, /ɡɒdˈzɪlə/; [ɡoꜜdʑiɾa] (About this soundlisten)) is a fictional monster, or kaiju, originating from a series of Japanese films. The character first appeared in the 1954 film Godzilla and became a worldwide pop culture icon, appearing in various media, including 32 films produced by Toho, four Hollywood films and numerous video games, novels, comic books and television shows. Godzilla has been dubbed the'King of the Monsters', a phrase first used in Godzilla, King of the Monsters! (1956), the Americanized version of the original film.",
+    tags: ["sea-monster", "reptile", "prehistoric"]
+)
```

Comment:
I decided to define these values in the test as opposed to make them `static let` properties of `NoteData` to keep them closer to where they're used.

Reply:
Fully agree. I initially thought of having these defined in a separate file, but it looks more natural to have them in the only file where they are used.